scripts/csv_import.py

```diff
@@ -19,9 +19,6 @@
 flags.DEFINE_string(
   "bedcounts_csv", None, "Path to csv file containing bedcounts data."
 )
-flags.DEFINE_string(
-  "timezone", "UTC", "Timezone in which dates should be interpreted."
-)
```

Comment:
I don't use this script so I don't really have an opinion about it.

Reply:
It should be ok, I introduced this option recently.